charts/brigade/values.yaml

```diff
@@ -177,6 +177,66 @@ worker:
 
   workspaceStorageClass: nfs
 
+logger:
+
+  linux:
+
+    image:
+      repository: krancour/brigade-logger-linux
```

Comment:
This triggered the thought: We can/should add logger image publishing to brigade.js (for linux) and GH actions (for windows) to Brigadecore's `brigadecore` DH org (and/or whichever registry we finalize on for v2).  Maybe a ticket to add/update CI (and update these references), if not filed already?

Reply:
This is actually a mistake-- that shouldn't say `krancour`, but `brigadecore`. I will fix that.

But stepping back, you're correct in observing that we're currently not publishing _any_ of the images we build. For the time being, with so few people actively working on this, building and deploying from source (see `make hack`) seems preferable to cluttering up whatever image repository we think this belongs in long-term.

I'd been thinking that we'd re-work our pipelines to publish "edge" images on merge to `v2` and semantically tagged images on release shortly before actually cutting the alpha release. Is that reasonable?

I will update that errant reference to `krancour`.

Reply:
> building and deploying from source (see make hack) seems preferable 

Indeed, during this period, this seems preferred.  Better to focus on v2 bread and butter.

> I'd been thinking that we'd re-work our pipelines to publish "edge" images on merge to v2 and semantically tagged images on release shortly before actually cutting the alpha release. Is that reasonable?

Sgtm, thanks!